.github/workflows/release.yml

```diff
@@ -7,40 +7,24 @@ on:
 
 jobs:
   goreleaser:
+    name: Build binary and Docker image
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code
+      - name: Checkout
         uses: actions/checkout@v2
+
       - name: Docker Login
         if: success() && startsWith(github.ref, 'refs/tags/')
         env:
           DOCKER_USERNAME: hellofreshtech
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         run: |
           echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin
+
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         if: success() && startsWith(github.ref, 'refs/tags/')
         with:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-  helm:
```

Comment:
Why are you removing this job?

Reply:
It doesn't work in release workflow properly, will be moved to separate workflow in next PR-s.